docs/cni-proposal.md

```diff
@@ -149,11 +149,12 @@ Here are the wiring steps to enable pod to pod communication:
 	ip netns exec ns1 arp -i veth-1c -s 169.254.1.1 <veth-1's mac> /* add static ARP entry for default gateway */
 	```
 
-* On host side, add host route so that incoming Pod's traffic can be routed to Pod.
+* On host side, add host route and routing rule so that incoming Pod's traffic can be routed to Pod.
 
 	```
 	/* Pod's IP address is 20.0.49.215 */
 	ip route add 20.0.49.215/32 dev veth-1 /* add host route */
+	ip rule add from all to 20.0.49.215/32 table main prio 512 /* add routing rule */
```

Comment:
1356 rule is added for from-pod rule, only need it when it is not primary ENI - https://github.com/aws/amazon-vpc-cni-k8s/blob/5bb9811e5b21701a2d119dd58ad49b83017decda/cmd/routed-eni-cni-plugin/driver/driver.go#L325. Maybe you can include both mentioning the difference as primary and secondary ENI.

Reply:
Sorry for delay.
I'll try to include the both messages :)

Reply:
@jayanthvn
I tried it, but how is it?
And also, I modified the example comment along the current behavior. e236a333c313ce4f7396c44426fbcbfa78f4b246